tests/core/test_memory.py

```diff
@@ -20,6 +20,43 @@
 from tests.base.models import ParityModuleRNN
 
 
+def almost_equals(a, b, rel_tol=0.0, abs_tol=0.0):
+    _almost_close = lambda a, b: abs(a - b) <= max(rel_tol * max(abs(a), abs(b)), abs_tol)
+    return _almost_close(a, b)
```

Comment:
You can use `torch.allclose` or `torch.testing.assert_allclose` for this

Reply:
oh nice will do it, im new to torch :P